vault/auth.go

```diff
@@ -347,6 +347,9 @@ func (c *Core) loadCredentials(ctx context.Context) error {
 			entry.Accessor = accessor
 			needPersist = true
 		}
+
+		// Sync values to the cache
+		entry.SyncCache()
```

Comment:
I think you need such a line in enableCredential as well (and in the secrets mount logic in the same place)

Reply:
Yea, thanks for catching that!

Reply:
Addressed in e1b17c6